Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```
@@ -997,8 +995,7 @@
           bounds: "-0.15,-0.3,0.15,0.3"
         hard: false
         mask:
-        - Impassable
-        - BulletImpassable
+        - Opaque # WWDP allow shooting through windows
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 10 but found 8

(indentation)